frontend/src/views/log/task/index.vue

```diff
@@ -73,19 +73,19 @@ const search = async () => {
     req.page = paginationConfig.currentPage;
     req.pageSize = paginationConfig.pageSize;
     loading.value = true;
-    await searchTasks(req)
-        .then((res) => {
-            loading.value = false;
-            data.value = res.data.items;
-            paginationConfig.total = res.data.total;
-        })
-        .catch(() => {
-            loading.value = false;
-        });
+    try {
+        const res = await searchTasks(req);
+        loading.value = false;
+        data.value = res.data.items;
+        paginationConfig.total = res.data.total;
+    } catch (error) {
+    } finally {
+        loading.value = false;
+    }
 };
 
 const openTaskLog = (row: Log.Task) => {
-    taskLogRef.value.openWithTaskID(row.id);
+    taskLogRef.value.openWithTaskID(row.id, !(row.status == 'Executing'));
 };
 
 onMounted(() => {
```

Comment:
This appears to be JavaScript, it could potentially contain HTML elements like `<div>`, `<span>` etc which can affect how the code is interpreted. In general, there's no major errors or issues but I'd recommend reviewing all parts of this file for clarity if it's part of a larger project.